nifi-nar-bundles/nifi-lumberjack-bundle/nifi-lumberjack-processors/src/main/java/org/apache/nifi/processors/lumberjack/event/LumberjackEvent.java

```diff
@@ -24,6 +24,7 @@
 /**
  * A Lumberjack event which adds the transaction number and command to the StandardEvent.
  */
+@Deprecated
```

Comment:
For all @Deprecated tags, please provide @deprecated Javadoc for why it is being deprecated and what they should switch to